exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestCTAS.java

```diff
@@ -357,6 +358,39 @@ public void testTableIsFoundWithinWorkspaceWhenNameStartsWithSlash() throws Exce
     }
   }
 
+  @Test
```

Comment:
These tests test Parquet Writer not something specific to CTAS, so I believe they should be moved to one of the classes in the `org.apache.drill.exec.physical.impl.writer` test package.